src/Ninja/G1CP/Content/Fixes/Session/fix149_DE_EN_ImprovedOreArmorName.d

```diff
@@ -0,0 +1,16 @@
+/*
+ * #149 Spelling - Improved Ore Armor (DE/EN)
+ */
+func int G1CP_149_DE_EN_ImprovedOreArmorName() {
+    var int symbId; symbId = MEM_GetSymbolIndex("ORE_ARMOR_H");
+    const string needle = "";
+    const string replace = "";
+    if (G1CP_Lang == G1CP_Lang_DE) {
+        needle  = "verbesserte Erzr�stung";
+        replace = "Verbesserte Erzr�stung";
+    } else if (G1CP_Lang == G1CP_Lang_EN) {
+        needle  = "Improved ore Armor";
+        replace = "Improved Ore Armor";
+    };
+    return (G1CP_ReplaceAssignStr(symbId, "C_Item.name", 0, needle, replace) > 0);
```

Comment:
Something that I now realize is that we should be careful here. If the language is neither English nor German, the instance function will be searched for `name = "";` and replaced with `name = "";`. Of course, this is not a problem and also not very likely to happen, but we should keep that in mind for future uses of `G1CP_ReplaceAssignStr` that the needle and replace string should be non-empty if possible.

Reply:
Thinking about it, an alternative implementation could have been
```D
func int G1CP_149_DE_EN_ImprovedOreArmorName() {
    var int symbId; symbId = MEM_GetSymbolIndex("ORE_ARMOR_H");
    const string needleEN  = "Improved ore Armor";
    const string replaceEN = "Improved Ore Armor";
    const string needleDE  = "verbesserte Erzrüstung"
    const string replaceDE = "Verbesserte Erzrüstung";
    var int count; count = 0;
    count += G1CP_ReplaceAssignStr(symbId, "C_Item.name", 0, needleEN , replaceEN);
    count += G1CP_ReplaceAssignStr(symbId, "C_Item.name", 0, needleDE , replaceDE);
    return (count > 0);
};
```
However, this is computationally less efficient.

For now, this fix is great as it is.

Reply:
We could instantiate the strings non-empty with highly unlikely content:

```D
    const string needle = "G1CP invalid string";
    const string replace = "G1CP invalid string";
```